modules/config/config.scm

```diff
@@ -94,4 +94,18 @@ end-of-c-declare
  (gambit-c (if (string=? (system-platform) "android") (##heartbeat-interval-set! -1.)))
  (else (if (string=? (system-platform) "android") (##set-heartbeat-interval! -1.))))
 
+(cond-expand
+ (android
+  (c-declare #<<EOF
+extern char* android_getFilesDir_info_get();
+char* android_getFilesDir_info()
+{
+ return android_getFilesDir_info_get();
+}
+extern char* android_getPackageCodePath();
+EOF
+)
+  (define (android-PackageCodePath) ((c-lambda () char-string "android_getPackageCodePath"))))
```

Comment:
This seems to be a strange naming conversion for a scheme function, more like Java with CamelCase?

Reply:
Also this function is not defined on non-android platforms, which may be a problem as how do we prevent users from including this in their code? Wouldn't it be safer to always declare it and simply not do anything on non-android platforms?